.promu.yml

```diff
@@ -1,4 +1,4 @@
-go: 1.5.4
+go: 1.6.2
```

Comment:
You will need to rebase your work given that this change and the one in `.travis.yml` are already in master.